CI/astyle/.astylerc

```diff
@@ -13,6 +13,7 @@ style=kr
 # 2 spaces, convert tabs to spaces
 indent=spaces=2
 convert-tabs
+lineend=linux
```

Comment:
I will not add this.
For those working on Windows, files in the working tree will be in CRLF so the astyle will failed.

Reply:
Ah, I see that the .gitattributes file indeed enables EOL translation/normalization for all text files. We could explicitly specify `eol=lf` as well, to checkout all files with lf as well (in practice, I think most tools on Windows do not really care about CRLF vs LF anymore, except maybe notepad). OTOH, with the current configuration, files should already be normalized to newlines by git on checkin, so this should usually be ok automatically.

One alternative would be to change the python script to apply lineend=linux only on non-Windows platforms and/or only in the CI-run, but that is probably too complicated and unexpected, so better leave that out.

I've removed the lineend commit again.

Reply:
I think about fforce LF on windows this would avoid any issue. We had the case in a PR, I guess user git config force to chieckin as CRLF

Reply:
Ah, in case we want that, I think it's a matter of adding `eol=lf` to all text lines in `.gitattributes`. But this can probably also be considered and tested separately of this PR, of course.

Reply:
Right. Done in #1021